Documentation/ApiOverview/Icon/Index.rst

```diff
@@ -79,3 +79,35 @@ You can also simple use the Fluid ViewHelper to render an icon in your view:
    {namespace core = TYPO3\CMS\Core\ViewHelpers}
    <core:icon identifier="my-icon-identifier" size="small" overlay="overlay-identifier" />
 
+.. _available-icons:
+
+Available Icons
+===============
+
+The TYPO3 Core comes with a number of icons that may be used in your extensions.
+
+To search for available icons, you can use one of these possibilities:
+
+Install Extension Styleguide
+----------------------------
+
+Install the extension *styleguide* as described in the Readme in the `installation
+<https://github.com/TYPO3/styleguide#installation>`__ section.
+
+Once, installed, you can view available icons by selecting help (?) on the top in the
+TYPO3 Backend, then *Styleguide* and then *Icons*, *All Icons*.
+
+There, browse through existing icons. Use the name under the icon (for example
+*actions-add*) as first parameter for *getIcon* in PHP or as value for
```

Comment:
Icon identifiers should use `:code:`, also `getIcon` should be `getIcon()` and have `:php:`, possibly also add the class name in front.